src/app/shared/components/products/products-list/products-list.component.ts

```diff
@@ -48,25 +48,22 @@ export class ProductsComponent {
       this.products = this.products.filter(
         (product) => deletedProd.id !== product.id
       );
-      this.productsService.isLoading = false;
       this.closeDeleteModal();
-    });
+    }, (err) => console.error(err));
   }
   addProduct(newProductData: Product): void {
     this.productsService.addProduct(newProductData).subscribe((data) => {
       this.products.push(data);
-      this.productsService.isLoading = false;
       this.closeAddProductModal();
-    });
+    }, (err) => console.error(err));
```

Comment:
You can remove these, since http errors are automatically reported to the console